public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -541,7 +601,7 @@ block dart-type-exceptions
 :marked
   ### Remember the brackets
   The brackets tell Angular to evaluate the template expression.
-  If we forget the brackets, Angular treats the string as a constant and *initializes the target property* with that string.
+  If directive forget the brackets, Angular treats the string as a constant and *initializes the target property* with that string.
```

Comment:
I think that was the first of a rogue find and replace that sprinkled "directive" in places "you" or "we" seems to have been. I think I changed all the rest of them. I just went back and changed this one and pushed up.